pkg/virt-handler/vm.go

```diff
@@ -1249,18 +1249,13 @@ func (d *VirtualMachineController) checkVolumesForMigration(vmi *v1.VirtualMachi
 			} else if err != nil {
 				return blockMigrate, err
 			}
-			blockMigrate = blockMigrate || !shared
 			if !shared {
-				return blockMigrate, fmt.Errorf("cannot migrate VMI with non-shared PVCs")
+				return true, fmt.Errorf("cannot migrate VMI with non-shared PVCs")
```

Comment:
Are we fixing a bug here? Makes sense to me, but would like to have @vladikr have a look at this.

Reply:
No, it's not a bug fix, the following part just confused me:
```golang
blockMigrate = blockMigrate || !shared
if !shared {
  return blockMigrate, fmt.Errorf("cannot migrate VMI with non-shared PVCs")
}
```
so I tried to simplify it:
if `!shared == true` then blockMigrate becomes `true` and we get to the return statement
if `!shared == false` then blockMigrate doesn't change and so the line `blockMigrate = blockMigrate || !shared` can be removed
and then we end up with just:
```golang
if !shared {
  return true, fmt.Errorf("cannot migrate VMI with non-shared PVCs")
}
```

Reply:
ah yes, sure... :+1: